tests/overlay_tests.cpp

```diff
@@ -203,8 +203,8 @@ class OverlayTest : public MesosTest
         "iptables -w -t nat -D POSTROUTING -s %s "
         "-m set --match-set %s dst "
         "-j MASQUERADE; "
-        "iptables -w -t filter -D DOCKER-ISOLATION "
-        "-j RETURN; "
+        "iptables -w -t filter -D DOCKER-ISOLATION -j RETURN; "
+        "iptables -w -t filter -D DOCKER-ISOLATION-STAGE-2 -j RETURN; "
```

Comment:
I don't see why we made a choice of `DOCKER-ISOLATION-STAGE-2` vs `DOCKER-ISOLATION-STAGE-1` here (or not both). 

Does that not matter?